lib/blumquist.rb

```diff
@@ -51,6 +55,13 @@ def resolve_json_pointers
     end
   end
 
+  def resolve_json_pointer(type_def)
```

Comment:
I don't understand the relation of `resolve_json_pointer` and `resolve_json_pointer!`. Would have expected that the former calls the latter but catches exceptions. Help 🤡